### PR TITLE
i18n(fr): Update `showcase.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Updated `showcase.mdx`

### CODE REVIEW

File: docs/src/content/docs/fr/showcase.mdx [hunk 1]
```diff
@@ -41,6 +41,11 @@ Les [modules d'extension](/fr/reference/plugins/) peuvent personnaliser la confi
 		title="starlight-links-validator"
 		description="Vérifier la validité des liens dans vos pages Starlight."
 	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-typedoc"
+		title="starlight-typedoc"
+		description="Générez des pages Starlight à partir de TypeScript en utilisant TypeDoc."
```
Comment (dreyfus92) [inline docs/src/content/docs/fr/showcase.mdx:47]:
```suggestion
		description="Générer des pages Starlight à partir de TypeScript en utilisant TypeDoc."
```

**Review by HiDeoo — APPROVED** [review]
Looks like everything is in order, thanks for the update :thumbsup: 🌟

**Comment by thomasbnt** [comment]
Aaarg just seen this opened PR 😆😵 ([Mine is same](https://github.com/withastro/starlight/pull/1309) with one edition on "Blogue" → "Blog")

**Review by thomasbnt — APPROVED** [review]
Good!

**Comment by HiDeoo** [comment]
> "Blogue" → "Blog"

"Blogue" is the translation [recommended](https://www.culture.fr/franceterme/terme/INFO870) by the official [French language enrichment commission](https://www.culture.fr/FranceTerme/Le-dispositif-d-enrichissement-de-la-langue-francaise) which has been used as reference when possible in the French translation of the Starlight docs.

**Comment by thomasbnt** [comment]
> > "Blogue" → "Blog"
> 
> "Blogue" is the translation [recommended](https://www.culture.fr/franceterme/terme/INFO870) by the official [French language enrichment commission](https://www.culture.fr/FranceTerme/Le-dispositif-d-enrichissement-de-la-langue-francaise) which has been used as reference when possible in the French translation of the Starlight docs _(Il arrive parfois qu'on garde les 2 termes quand la version française est très différente et peu utilisée mais ca ne serait pas le cas ici)_.

Ah oui j'ai vu ça, c'est parce qu'on utilise quasi systématiquement le terme anglais "Blog" que ça m'a paru un peu étrange. Ok!

**Pull request merged**